### PR TITLE
Button, Card, Image component

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Added buttons, card and image components

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
+.btn{
+    line-height: 1.5;
+    border: 1px solid transparent;
+    cursor: pointer;
+    padding: 6px 10px;
```
Comment (AshutoshM29) [inline components/button.css:5]: You should try using rem

File: components/button.css [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    font-size: 1.5rem;
+    font-weight: 500;
+    border-radius: 4px;
+    color: black;
+    background-color: white;
+    transition: all ease-in-out 0.15s;
+}
+
+.btn-primary{
+    color: #fff;
```
Comment (AshutoshM29) [inline components/button.css]: You can use var here too

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+.btn-outline-danger{
+    color: var(--btn-color-danger);
+    border-color: var(--btn-color-danger);
+}
+
+.btn-outline-danger:hover{
+    background-color: var(--btn-color-danger);
+    color: white;
```
Comment (AshutoshM29) [inline components/button.css:132]: Same here, you can use var here as well

**Comment by mmanjuprasad** [comment]
In the card section u have provided a carbon iframe with no code inside 
Please check and rectify it
The  image section also has no code in it
image
image

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Work on the suggested things, the rest looks good to merge!!

**Comment by mmanjuprasad** [comment]
The code looks fine apart from changes suggested from ashutosh
can be merged after suggested changes

File: components/button.css (hunk 2 above)
Comment (kaustubh-jsr) [reply components/button.css]: Done

File: components/button.css (hunk 1 above)
Comment (kaustubh-jsr) [reply components/button.css:5]: I used px for measures less than or equal to 10px, as was suggested in the session. Still will think about this

File: components/button.css (hunk 3 above)
Comment (kaustubh-jsr) [reply components/button.css:132]: Done

**Pull request merged**